Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.yml

```diff
@@ -0,0 +1,253 @@
+    just send alerts to demisto, if you flag this checkbox we also send incidents.
+    When a fetch integration was configured without incidents, if enabled means that
+    the new n2os incidents will be sent to XSOAR, but not the old one.
+  display: Fetch also Nozomi incidents
+  hidden: false
+  name: fecthAlsoIncidents
+  required: false
+  type: 8
+description: |-
+  The Nozomi Networks Guardian platform is a hardware or virtual appliance that is used to monitor OT/IoT/IT networks. It combines asset discovery, network visualization, vulnerability assessment, risk monitoring and threat detection in a single solution.
+    This integration is used to gather alert and asset information from Nozomi.
+display: Nozomi Networks
+name: Nozomi Networks
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: You can add a filter to get exactly the assets you want. For example
+        'where ip match 10.0.1.10', 'where vendor ==  Selta Telematica S.p.a'
+      isArray: false
+      name: filter
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '500'
+      description: Maximun number of assets get from Nozomi, can't be bigger than
+        1000
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: This command permits you to get some assets from Nozomi, you can
+      use the query filter to to refine your search. With the limits you can decide
+      the max number of assets you can retrieve from Nozomi, the limit can't be bigger
+      than 1000.
+    execution: false
+    name: nozomi-find-assets
+    outputs:
+    - contextPath: Nozomi.Asset.id
+      description: uniq id of an asset
+      type: String
+    - contextPath: Nozomi.Asset.level
+      description: network layer
+      type: Number
+    - contextPath: Nozomi.Asset.capture_device
+      description: source from which the asset was captured
+      type: String
+    - contextPath: Nozomi.Asset.ip
+      description: array of asset ip
+      type: Unknown
+    - contextPath: Nozomi.Asset.mac_address
+      description: array of asset mac address
+      type: Unknown
+    - contextPath: Nozomi.asset.mac_vendor
+      description: array of mac vendor
+      type: Unknown
+    - contextPath: os
+      description: operating system
+      type: String
+    - contextPath: vendor
+      description: asset vendor
+      type: String
+    - contextPath: Nozomi.Asset.firmware_version
+      description: firmaware version
+      type: String
+    - contextPath: serial_number
+      description: serial number
+      type: String
+    - contextPath: product_name
+      description: product name
+      type: String
+    - contextPath: type
+      description: asset type as 'OT_device'
+      type: String
+    - contextPath: protocols
+      description: array of asset protocols
+      type: Unknown
+  - arguments:
+    - default: false
+      defaultValue: '[]'
```

Comment:
remove

Reply:
all the four lines?
Sorry I dont understand :)